.github/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Release 0.2.5 (development release)
+
+<h3>New features since last release</h3>
+
+<h3>Improvements</h3>
+
+<h3>Breaking changes</h3>
+
+<h3>Bug fixes</h3>
+
+* Templates are no longer changed when calling them and replacing the parameters with actual values.
+
+  ```python
+  bb_program = bb_template(**template_variables)
+  ```
```

Comment:
(I would remove the code block, since it doesn't really show anything)

```suggestion
* Templates are no longer changed when calling them and replacing the parameters with actual values.
  [(#34)](https://github.com/XanaduAI/blackbird/pull/3)
```